app/controllers/auth/sessions_controller.rb

```diff
@@ -112,7 +113,7 @@ def set_instance_presenter
   def home_paths(resource)
     paths = [about_path]
     if single_user_mode? && resource.is_a?(User)
-      paths << short_account_path(username: resource.account)
+      paths << short_account_path(resource.account)
```

Comment:
Can we just use `TagManager.instance.url_for` here so we don't have to include `RoutingHelper`?